--- vidos_base_config
+++ vidos_avc_config
@@ -3 +3 @@
-BR2_TOOLCHAIN_EXTERNAL_BOOTLIN_X86_64_MUSL_STABLE=y
+BR2_TOOLCHAIN_EXTERNAL_BOOTLIN_X86_64_GLIBC_STABLE=y
@@ -12,0 +13 @@
+BR2_ROOTFS_POST_BUILD_SCRIPT="board/vidos_av1/remove_firmware.sh"
@@ -14,0 +16 @@
+BR2_ROOTFS_POST_SCRIPT_ARGS="avc"
@@ -26,0 +29 @@
+BR2_PACKAGE_FFMPEG_EXTRACONF="--disable-everything --enable-libopenh264 --enable-libfdk_aac --enable-decoder='libopenh264,libfdk_aac' --enable-parser='h264,aac' --enable-demuxer='matroska,mov' --enable-protocol=file --enable-outdev='fbdev,alsa,oss' --enable-filter=aresample"
@@ -27,0 +31 @@
+BR2_PACKAGE_FDK_AAC=y
@@ -32,0 +37 @@
+BR2_PACKAGE_LIBOPENH264=y
