--- vidos_base_config
+++ vidos_webm_config
@@ -10 +10 @@
-BR2_TARGET_GENERIC_HOSTNAME="VidOS_v2.00_base"
+BR2_TARGET_GENERIC_HOSTNAME="VidOS_v2.00_webm"
@@ -18,0 +19 @@
+BR2_ROOTFS_POST_SCRIPT_ARGS="webm"
@@ -30,0 +32 @@
+BR2_PACKAGE_FFMPEG_EXTRACONF="--disable-everything --enable-decoder='vp8,vp9,opus' --enable-parser='vp8,vp9,opus' --enable-demuxer=matroska --enable-protocol=file --enable-outdev='fbdev,alsa,oss'"
